pkg/api/common/versions.go

```diff
@@ -98,8 +98,9 @@ var AllKubernetesSupportedVersions = map[string]bool{
 	"1.12.1":         false,
 	"1.12.2":         false,
 	"1.12.4":         false,
-	"1.12.5":         true,
+	"1.12.5":         false,
 	"1.12.6":         true,
+	"1.12.7":         true,
```

Comment:
I think you forgot to add it to the vhd script

Reply:
Done @mboersma FYI